docs/flocker-features/gce-configuration.rst

```diff
@@ -0,0 +1,93 @@
+.. _gce-dataset-backend:
+
+=====================================
+Google GCE / PD Block Device Backend 
+=====================================
+
+.. begin-body
+
+The GCE backend uses PD volumes as the storage for datasets.
+This backend can be used when Flocker dataset agents are run on GCE instances.
+The configuration item to use GCE should look like:
```

Comment:
:question: Perhaps instead. "The `agent.yml` configuration file for the GCE backend should contain the following:"